docs/data-masking-component-functions.md

```diff
@@ -0,0 +1,388 @@
+
+# Data masking component functions
+
+
+
+
+
+The general purpose data masking functions are the following:
```

Comment:
Shouldn't we just refer to upstream docs, and only describe the differences in our implementation? It's mostly the same thing.

Reply:
We write complete sets of documentation.